content/source/docs/cloud/integrations/service-now/service-catalog.html.md

```diff
@@ -0,0 +1,160 @@
+      organization's VCS providers settings. This ID specifies which VCS
+      provider configured in Terraform Cloud hosts the desired repository.
+    - Identifier: The VCS repository that contains the Terraform configuration
+      for this workspace template. Repository identifiers are determined by your
+      VCS provider; they typically use a format like
+      `<ORGANIZATION>/<REPO_NAME>` or `<PROJECT KEY>/<REPO_NAME>`. Azure DevOps
+      repositories use the format `<ORGANIZATION>/<PROJECT>/_git/<REPO_NAME>`.
+    - The remaining fields are optional.
+      - Branch: The branch within the repository, if different from the default
+        branch.
+      - Working Directory: The directory within the repository containing
+        Terraform configuration.
+      - Terraform Version: The version of Terraform to use. This will default to
+        the latest version of Terraform supported by your Terraform Cloud
+        instance.
+1. Click "Submit".
+
+![Screenshot: ServiceNow New VCS Repository](./images/service-now-vcs-repository.png "Screenshot of the ServiceNow Terraform New VCS Repository page")
+
+## Configure a Variable Set
+
+Most Terraform configuration can be customized with Terraform variables or
+environment variables. You can create a Variable Set within ServiceNow to
+contain the variables needed for a set of configuration. Your Terraform Admin
+should provide these to you.
+
+1. In ServiceNow, open the "Service Catalog > Variable Sets" table by searching for
+   "variable sets" in the left-hand navigation.
+1. Click "New" to add a Variable Set.
+1. Select "Single-Row Variable Set".
+    - Title: User-visible title for the variable set.
+    - Internal name: The internal name for the variable set.
+    - Order: The order in which the variable set will be displayed.
+    - Type: Should be set to "Single Row"
+    - Application: Should be set to "Terraform"
+    - Display title: Whether the title is displayed to the end user.
+    - Layout: How the variables in the set will be displayed on the screen.
+    - Description: A long description of the variable set.
+1. Click "Submit" to create the variable set.
+1. Find and click on the Title of the new variable set in the Variable Sets
+   table.
+1. At the bottom of the variable set details page, click "New" to add a new
+   variable.
+  - Type: Should be "Single Line Text" for most variables, or "Masked" for
+    variables containing sensitive values.
+  - Question: The user-visible question or label for the variable.
+  - Name: The internal name of the variable. This must match the name of the
+      Terraform or environment value. Consult the table below to determine the
+      proper prefix for each variable name.
+  - Tooltip: A tooltip to display for the variable.
+  - Example Text: Example text to show in the variable's input box.
+1. Under the "Default Value" tab, you can set a default value for the variable.
```

Comment:
Isn't this part of the previous step? Or nah?

Reply:
Yeah, it's a little awkward, since the previous list is just bullets, but I think it makes sense in context? Or I could put the entire "make a variable" steps under another level that says "Make as many variables as you need to", but I don't like making super nested lists either.